README.markdown

```diff
@@ -1149,6 +1149,7 @@ Directives
 * [lua_socket_keepalive_timeout](#lua_socket_keepalive_timeout)
 * [lua_socket_log_errors](#lua_socket_log_errors)
 * [lua_ssl_ciphers](#lua_ssl_ciphers)
+* [lua_ssl_ciphersuites](#lua_ssl_ciphersuites)
```

Comment:
It seems the Nginx way to solve this problem is to use `ssl_conf_command`, see:
https://trac.nginx.org/nginx/ticket/1529

What about using lua_ssl_conf_command instead?

Reply:
ok, it seems to be a more general and consistent way. I'll open another PR to add lua_ssl_conf_command, and close this one

Reply:
#1890 a new PR has been opened to substitute this one